pkg/minishift/registration/utils.go

```diff
@@ -30,7 +31,7 @@ type RegistrationParameters struct {
 }
 
 // Register host VM
-func RegisterHostVM(host *host.Host, param *RegistrationParameters) error {
+func RegisterHostVM(host *host.Host, param *RegistrationParameters, proxyInfo string) error {
```

Comment:
If we do as per your suggestion then it be placed in the common flags section right? Then it would be an issue like https://github.com/minishift/minishift/issues/432